deployment/articles/05-upgrading-to-liferay-7-2/06-upgrading-the-liferay-database/01-upgrading-the-liferay-database-intro.markdown

```diff
@@ -25,15 +25,19 @@ If you are upgrading from Liferay Portal 6.2 or earlier, it's recommended to use
 the upgrade tool to upgrade everything. It's the easiest, most comprehensive way
 to upgrade from those versions. Since version 7.0, however, @product@'s modular
 framework lets you upgrade modules--even the core--individually. Focusing first
-on upgrading the core and your most important modules might be better for you.
-The point is, the @product-ver@ upgrade process is flexible. 
+on upgrading the core and your most important modules might be advisable for
+big databases so that you can make a backup in the middle of the process. The
+point is, the @product-ver@ upgrade process is flexible.
 
 **Note:** Liferay enterprise subscribers can use the upgrade tool to execute 
 upgrades for fix packs. Since @product@ 7.1, a fix pack's micro upgrade
-processes (core database schema micro version changes) are not mandatory. This
-means you can install a fix pack (i.e., core code) without having to execute the
-database schema micro version changes---you can execute micro version changes
-when you want, even outside of a major or minor version upgrade. Before using
-the upgrade tool to execute a fix pack's micro upgrade process, however, you
-must shut down the server, install the fix pack, and 
+processes in the Core (database schema micro version changes) are not
+mandatory. This means you can install a fix pack (i.e., core code) without
+having to execute the database schema micro version changes---you can execute
+micro version changes when you want, even outside of a major or minor version
+upgrade. Before using the upgrade tool to execute a fix pack's micro upgrade
+process, however, you must shut down the server, install the fix pack, and 
 [back up the @product@ database, installation, and Document Library store](/discover/deployment/-/knowledge_base/7-1/backing-up-a-liferay-installation). 
+
+For modules, micro database schema version changes are executed after fix pack
+installation unless you set AutoUpgrade config to false.
```

Comment:
@achaparro Will module micro database schema version changes in fix packs execute automatically after fix pack installation unless AutoUpgrade is set to false?

Thanks!

Reply:
Yes, that's it!

Reply:
The upgrade is executed on server startup